apps/backend/src/websocket/server.py

```
@@ -0,0 +1,215 @@
+                }))
+                
+                logger.info(f"WebSocket client {client_id} authenticated as {user_id}")
+            else:
+                await websocket.send_text(json.dumps({
+                    "type": MessageType.ERROR,
+                    "error_code": "AUTH_FAILED", 
+                    "message": "Invalid authentication token"
+                }))
+                return
+                
+        except asyncio.TimeoutError:
```

Ruff (0.12.2)

Replace aliased errors with `TimeoutError`

Replace `asyncio.TimeoutError` with builtin `TimeoutError`

(UP041)